src/gui/plugins/component_inspector/Light.qml

```diff
@@ -285,6 +289,45 @@ Rectangle {
         id: grid
         width: parent.width
 
+        RowLayout {
+          Rectangle {
+            color: "transparent"
+            height: 40
+            Layout.preferredWidth: isOnText.width + indentation*3
+            Loader {
```

Comment:
You can remove this `Loader`. I don't think plotting the light state is valueable.